src/Form/Type/ChoiceFieldMaskType.php

```diff
@@ -59,16 +62,14 @@ public function configureOptions(OptionsResolver $resolver): void
     }
 
     /**
-     * @return string
-     *
-     * @phpstan-return class-string<FormTypeInterface>
+     * @phpstan-return class-string<FormTypeInterface>|null
      */
-    public function getParent()
+    public function getParent(): ?string
```

Comment:
```suggestion
    public function getParent(): string
```

Reply:
https://github.com/symfony/symfony/blob/7cb8ba585dcb4243c4245487281654df149bd3ad/src/Symfony/Component/Form/FormTypeInterface.php#L78

Same for all the other comments.

Reply:
In Symfony5 `FormTypeInterface` have not return type hints. Adding it is BC-break, so mybe they will be added in Symfony6. In sonata code we always return string. Even when we add support for Symfony6 and change 'string' to `?string` it will be BC becouse we will be still return string. 

Also:
http://sandbox.onlinephpfunctions.com/code/5548d6e1173bb5c4e830a72d175ac8e439b72d6d

Reply:
Fixed